test/Microsoft.AspNet.Mvc.FunctionalTests/PrecompilationTest.cs

```diff
@@ -32,59 +22,6 @@ public PrecompilationTest(MvcTestFixture<PrecompilationWebSite.Startup> fixture)
 
         public HttpClient Client { get; }
 
-        [ConditionalFact]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
-        public async Task PrecompiledView_RendersCorrectly()
-        {
-            // Arrange
-            IServiceCollection serviceCollection = null;
-            var server = TestHelper.CreateServer(_app, SiteName, services =>
-            {
-                _configureServices(services);
-                serviceCollection = services;
-            });
-            var client = server.CreateClient();
-
-            var serviceProvider = serviceCollection.BuildServiceProvider();
-            var applicationEnvironment = serviceProvider.GetRequiredService<IApplicationEnvironment>();
-
-            var viewsDirectory = Path.Combine(applicationEnvironment.ApplicationBasePath, "Views", "Home");
-            var indexContent = File.ReadAllText(Path.Combine(viewsDirectory, "Index.cshtml"));
-
-            // We will render a view that writes the fully qualified name of the Assembly containing the type of
-            // the view. If the view is precompiled, this assembly will be PrecompilationWebsite.
-            var assemblyNamePrefix = GetAssemblyNamePrefix();
-
-            try
-            {
-                // Act - 1
-                var response = await client.GetAsync("http://localhost/Home/Index");
```

Comment:
Didn't look to me like the test demonstrated anything of interest given other tests in this class. But if you like the particular view, sure I'll restore. Are you that excited?